src/Info.ts

```diff
@@ -133,22 +217,55 @@ export default class Info {
         return `Successfully removed ${command}`;
     }
 
-    private listInfo() {
-        const index = this.infos;
+    private async listInfo(message: Discord.Message) {
+
+        const maxLength = 80;
 
-        let message = `The available info commands are: \n`;
+        let firstPage: Discord.RichEmbed | null = null;
+        let pages: { [emoji: string]: Discord.RichEmbed} = {};
+        for (const category of this.categories) {
+            const categoryItems = this.infos.filter(i => i.categoryId == category.icon);
+            
+            const page = new Discord.RichEmbed();
+            page.setTitle(category.explanation);
+
+            for (const item of categoryItems) {
+                const message = item.message.length > maxLength ? item.message.substr(0, maxLength - 3) + "..." : item.message;
+                page.addField("!note " + item.command, message, false);
+            }
 
-        for (const info of this.infos) {
-            message += `- \`!${this.command} ${info.command}\`\n`;
+            if (!firstPage) firstPage = page;
+            pages[category.icon] = page;
         }
 
-        return message;
+        let reply = await message.channel.send({ embed: firstPage });
+        if (reply instanceof Array) reply = reply[0];
```

Comment:
`Array.isArray`

Reply:
Unnecessary change imo, both are equally readable

Reply:
This is better practice, since `instanceof Array` may fail in certain contexts (most often in cross-document browser settings). While it indeed doesn't matter in Node, it is a good habit to get into ;)